algorithms/heap/k_closest_points.py

```diff
@@ -0,0 +1,44 @@
+"""Given a list of points, find the k closest to the origin.
+
+Idea: Maintain a max heap of k elements.
+We can iterate through all points.
+If a point p has a smaller distance to the origin than the top element of a heap, we add point p to the heap and remove the top element.
+After iterating through all points, our heap contains the k closest points to the origin.
+"""
+
+
+from heapq import *
```

Comment:
```suggestion
from heapq import heapify, heappushpop
```
or 
```suggestion
import heapq as hq
```
Because importing everything is generally not a good practice